packages/clerk-js/src/ui/components/SignUp/SignUpSocialButtons.tsx

```diff
@@ -10,7 +10,9 @@ import { SocialButtons } from '../../elements/SocialButtons';
 import { useNavigate } from '../../hooks';
 import { handleError } from '../../utils';
 
-export const SignUpSocialButtons = React.memo((props: SocialButtonsProps) => {
+export type SignUpSocialButtonsProps = SocialButtonsProps & { continueSignUp?: boolean };
```

Comment:
❓ why didn't we add `continueSignUp?: boolean` to `SocialButtonsProps` type in `elements/SocialButtons` ?

Reply:
Because it doesn't apply to sign-in.